ts/parts/Tooltip.ts

```diff
@@ -728,7 +728,7 @@ class Tooltip {
                     container,
                     0,
                     0,
-                    {},
+                    { ...this.chart.options.chart?.style },
```

Comment:
I think we don't need this - simple `this.chart.options.chart?.style` should be enough? And we won't create extra polyfill.

Reply:
You're right, fix incoming.